proposals/lip-0045.md

```diff
@@ -1157,7 +1071,7 @@ Let `genesisBlockAssetBytes` be the `data` bytes included in the block assets fo
 * Within each substore, check that all entries have a different `storeKey`.
 * For each entry in the chain data substore there must be a corresponding entry (with the same value of `storeKey`) in the outbox root, channel data, and chain validators substores and viceversa. However, if the chain account has status set to terminated, the entry in the outbox root must *not* be present.
```

Comment:
This dependency somehow suggest that the chain data, channel data and chain validators substores could be merged into one.

Reply:
I'd prefer to keep it as it is.

Reply:
Okay, all good.